app/src/main/java/com/infomaniak/swisstransfer/ui/components/IllustratedMessageBlock.kt

```diff
@@ -0,0 +1,89 @@
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.swisstransfer.ui.components
+
+import android.content.res.Configuration
+import androidx.annotation.StringRes
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import com.infomaniak.swisstransfer.R
+import com.infomaniak.swisstransfer.ui.images.AppImages.AppIllus
+import com.infomaniak.swisstransfer.ui.images.illus.MascotSearching
+import com.infomaniak.swisstransfer.ui.theme.Dimens
+import com.infomaniak.swisstransfer.ui.theme.Margin
+import com.infomaniak.swisstransfer.ui.theme.SwissTransferTheme
+
+@Composable
+fun IllustratedMessageBlock(
+    icon: ImageVector,
+    @StringRes title: Int,
+    description: String,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier,
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        Image(imageVector = icon, contentDescription = null)
+
+        Spacer(Modifier.height(Margin.XLarge))
+
+        Text(
+            text = stringResource(title),
+            textAlign = TextAlign.Center,
+            style = SwissTransferTheme.typography.h1,
+        )
+
+        Spacer(Modifier.height(Margin.Medium))
+
+        Text(
+            text = description,
+            textAlign = TextAlign.Center,
+            style = SwissTransferTheme.typography.bodyRegular,
+            color = SwissTransferTheme.colors.secondaryTextColor,
+            modifier = Modifier.widthIn(max = Dimens.DescriptionWidth),
+        )
+    }
+}
+
+@Preview(name = "Light mode")
+@Preview(name = "Dark mode", uiMode = Configuration.UI_MODE_NIGHT_YES or Configuration.UI_MODE_TYPE_NORMAL)
+@Composable
+private fun IllustratedMessageBlockPreview() {
+    SwissTransferTheme {
+        Surface {
+            IllustratedMessageBlock(
+                icon = AppIllus.MascotSearching,
+                title = R.string.noTransferReceivedTitle,
+                description = "All transfers you receive over the last 30 days will appear here.",
```

Comment:
I didn't spot that one earlier but you can still call 
```suggestion
                description = stringResource(R.string.noTransferReceivedDescription),
```